### PR TITLE
support addon deployment config

### PR DESCRIPTION
**Pull request by skeeey**
Signed-off-by: Wei Liu <liuweixa@redhat.com>

### CODE REVIEW

File: charts/managed-serviceaccount/templates/clustermanagementaddon.yaml [hunk 1]
```diff
@@ -6,3 +6,6 @@ spec:
   addOnMeta:
     displayName: managed-serviceaccount
     description: managed-serviceaccount
+  supportedConfigs:
```
Comment (qiujian16) [inline charts/managed-serviceaccount/templates/clustermanagementaddon.yaml:9]: do we need to add this by default? or we have a variable on chart to decide.

Comment (skeeey) [reply charts/managed-serviceaccount/templates/clustermanagementaddon.yaml:9]: yeah, put a variable on chart may be a better way

Comment (skeeey) [reply charts/managed-serviceaccount/templates/clustermanagementaddon.yaml:10]: add `enableAddOnDeploymentConfig ` to control this

**Pull request merged**